### DISCUSSION TITLE
Specify api level while building packages

### DISCUSSION THREAD

**Discussion by John4650-hub** [termux/termux-packages#22437] [Q&A] [answered]
I know that i can specify the api level in build.sh file of a package using TERMUX_PKG_API_LEVEL
How can i specify the API level while building   multiple packages at once without modifying the packages' build.sh scripts

**Comment by agnostic-apollo** [comment, marked as answer]
Currently, no command line option exists, you can change default value.

https://github.com/termux/termux-packages/blob/3378db1e7de804522d3d6efe625b14ffc6270515/scripts/build/termux_step_setup_variables.sh#L10

**Reply by John4650-hub** [reply]
Thank you

**Reply by John4650-hub** [reply]
@agnostic-apollo  will this take effect while building bootstrap package?

**Reply by agnostic-apollo** [reply]
yes for `build-bootstrap.sh`.
